src/rshim.c

```diff
@@ -220,13 +220,15 @@ bool rshim_has_pcie_reset_delay = false;
 int rshim_pcie_enable_vfio = 1;
 int rshim_pcie_enable_uio = 1;
 int rshim_pcie_intr_poll_interval = 10;  /* Interrupt polling in milliseconds */
-int rshim_force_mode = 0;  /* Request ownship when entering drop mode */
+int rshim_force_mode = 0;  /* always keep /dev/rshim<N> & send a force cmd */
 
 /* Array of devices and device names. */
 rshim_backend_t *rshim_devs[RSHIM_MAX_DEV];
 char *rshim_dev_names[RSHIM_MAX_DEV];
 char *rshim_blocked_dev_names[RSHIM_MAX_DEV];
 
+bool rshim_force_cmd_pending[RSHIM_MAX_DEV];
```

Comment:
Add a comment for this one?

By the way, you're changing FORCE_CMD to FORCE_MODE. Would it make more sense to use 'rshim_force_mode_pending' here?  Just a suggestion, but not important.

Reply:
It's different and explained in rshim.conf and rshim.8.

FORCE_MODE means 1) keeping /dev/rshim<N>/misc and 2) sending out a Force command.

Will add comment here.